### ISSUE TITLE
V1.9.0 CORE LIGHTNING turns to "Not Started | Not Ready Yet | No Data" after a few hours

### ISSUE THREAD

**Issue by dan-str** [raspiblitz/raspiblitz#3884]
The first few hours node is running fine and working.
After app. 6 hours CLN stops. The failure was repeated multiple times already on this node.

The status screen then says:

Blocks xxxxxy/xxxxxy Sync OK 100.00%
   
CLN v23.02.2 
Not Started | Not Ready Yet | No Data

![Bildschirmfoto vom 2023-05-30 16-00-50](https://github.com/rootzoll/raspiblitz/assets/106260390/60a006af-e2d1-4a56-a2ae-a2879157aa36)

When returning to the menu it shows Blockchain 100% and  Lightning 100%.

![Bildschirmfoto vom 2023-05-30 16-01-17](https://github.com/rootzoll/raspiblitz/assets/106260390/86f370a6-20d2-428a-a544-d12dd35798a3)

Debug -l:
https://l.termbin.com/fmji

 I saw the issue #3879 and #3882 but they are with LND. This one is with CLN.

**Comment by wchancao** [comment]
You run out of space on your sd card... Thats why your cln backup plugin says theres an error writing in the database

**Comment by dan-str** [comment]
The actual SD card is a 32 GB Samsung PRO Endurance (like recommended). I will try a bigger one and post the result here.

**Comment by dan-str** [comment]
Looks good with a bigger SD-Card. I now tried a SanDisk Ultra 128GB microSDXC and it's running.
But question is, what can be removed from the 32GB SD card to make it run again? Or does the recommendation needs to be changed to bigger SD cards?

**Comment by dan-str** [comment]
![grafik](https://github.com/rootzoll/raspiblitz/assets/106260390/2850ad19-6bf5-405d-8ec9-82f76cf35343)

**Comment by surfac3** [comment]
i think this issue is connected to hsm secret gets encrypted .. do you by any chance get this error? in the logs? 

Did the CLN node by any chance stop with error in the log like this one? 
2023-05-27T17:43:36.445Z DEBUG   hsmd: pid 2555751, msgfd 48
2023-05-27T17:43:36.501Z DEBUG   hsmd: capability +WIRE_HSMD_CHECK_PUBKEY
2023-05-27T17:43:36.501Z DEBUG   hsmd: capability +WIRE_HSMD_SIGN_ANY_DELAYED_PAYMENT_TO_US
2023-05-27T17:43:36.695Z INFO    plugin-backup.py: root        : Comparing backup version 3858405 versus first write version 3858406
2023-05-27T17:43:36.696Z INFO    plugin-backup.py: Comparing backup version 3858405 versus first write version 3858406
2023-05-27T17:43:36.696Z INFO    plugin-backup.py: root        : Versions match up
2023-05-27T17:43:36.696Z INFO    plugin-backup.py: Versions match up

**Comment by surfac3** [comment]
In my case i wasnt able to get to the bottom of the issue .. all i foundout is that reboot of the node fixed the issue for few hours but in my case it was connected to hsm secret being locked

**Comment by BitromNO** [comment]
CLN sync on new node with proxmox is on day 3 now. 98%.. not that slow last time i checked, a friend of mine gave up and removed CLN..

**Comment by dan-str** [comment]
![grafik](https://github.com/rootzoll/raspiblitz/assets/106260390/61fda4e8-f3b6-4e0f-b1a5-c4c9a50d26f8)

It's also filling the 128 GB SD-card after some days, again same failure.

**Comment by dan-str** [comment]
> In my case i wasnt able to get to the bottom of the issue .. all i foundout is that reboot of the node fixed the issue for few hours but in my case it was connected to hsm secret being locked

Yep, I guess there is more than one problem.

![grafik](https://github.com/rootzoll/raspiblitz/assets/106260390/33c8b0b4-d0d3-4016-ae99-750004ce77b4)

**Comment by openoms** [comment]
we will need to find which file is growing to be able to take care of it.

Can list the files bigger than 500MB (eith the `/mnt/hdd` dir excluded):
```
sudo find / -type d \( -path "/mnt/hdd" -o -path "/mnt/hdd/*" \) -prune -o -type f -size +500M -exec ls -lh {} \;
```
Or use the tool ncdu for a graphical file browser:
```
# install
sudo apt-get install ncdu
# run, starting in the root directory
sudo ncdu /
```

**Comment by dan-str** [comment]
> we will need to find which file is growing to be able to take care of it.
> 
> Can list the files bigger than 500MB (eith the `/mnt/hdd` dir excluded):
> 
> ```
> sudo find / -type d \( -path "/mnt/hdd" -o -path "/mnt/hdd/*" \) -prune -o -type f -size +500M -exec ls -lh {} \;
> ```
> 
> Or use the tool ncdu for a graphical file browser:
> 
> ```
> # install
> sudo apt-get install ncdu
> # run, starting in the root directory
> sudo ncdu /
> ```

Does that help?

sudo find / -type d \( -path "/mnt/hdd" -o -path "/mnt/hdd/*" \) -prune -o -type f -size +500M -exec ls -lh {} \;
find: ‘/proc/1746294/task/1746294/fdinfo/5’: No such file or directory
find: ‘/proc/1746294/fdinfo/6’: No such file or directory
find: ‘/proc/1747689’: No such file or directory
find: ‘/proc/1748082’: No such file or directory
find: ‘/proc/1748119’: No such file or directory
find: ‘/proc/1748120’: No such file or directory
find: ‘/proc/1748121’: No such file or directory
find: ‘/proc/1748122’: No such file or directory
find: ‘/proc/1748124’: No such file or directory
find: ‘/proc/1748127’: No such file or directory
find: ‘/proc/1748128’: No such file or directory
find: ‘/proc/1748129’: No such file or directory


In the meanwhile I deleted CLN and installed LND instead, so I guess I cannot help anymore...

**Comment by wisegorilla** [comment]
@openoms @dan-str On my install, the issue was two log files in /var/log growing very quickly, filling with constant errors that appeared to be related to the Blitz API/WebUI.  I disabled that option in the console menu and the issue went away.  I can try to re-enable and get those errors again if that would help reproduce the issue.

**Comment by giybs372** [comment]
> @openoms @dan-str On my install, the issue was two log files in /var/log growing very quickly, filling with constant errors that appeared to be related to the Blitz API/WebUI. I disabled that option in the console menu and the issue went away. I can try to re-enable and get those errors again if that would help reproduce the issue.

I can't even get into the console menu now - it keeps stopping at 100% on both - I've reflashed and installed again but no different, it has synced a couple times but after a while it always goes back to the 100% progress with 16 or 17 peers. the LCD shows "Not Started | Not Ready Yet | No Data"

**Comment by cstenglein** [comment]
Most likely also related to #3879

**Comment by rootzoll** [comment]
OK can you test if the available v1.10rc2 is fixing this - download here: https://github.com/raspiblitz/raspiblitz/tree/v1.10#downloading-the-software

**Comment by psalzAppDev** [comment]
> @openoms @dan-str On my install, the issue was two log files in /var/log growing very quickly, filling with constant errors that appeared to be related to the Blitz API/WebUI. I disabled that option in the console menu and the issue went away. I can try to re-enable and get those errors again if that would help reproduce the issue.

Thanks, disabling Blitz API and WebUI seems to have fixed this issue for me. (Raspiblitz v1.9.0)
